### PR TITLE
[stable/prometheus-postgres-exporter] add ability to pass additional container arguments such as log level

### PR DESCRIPTION
**Pull request by yashbhutwala**
Signed-off-by: Yash Bhutwala <ymb002@bucknell.edu>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by yashbhutwala** [comment]
@sameersbn @prydonius can you please take a look?

File: stable/prometheus-postgres-exporter/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.8.0"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 1.3.0
+version: 1.3.1
```
Comment (cpanato) [inline stable/prometheus-postgres-exporter/Chart.yaml]:
```suggestion
version: 1.4.0
```
this pr is adding some new feature for the chart not a bug fix

Comment (yashbhutwala) [reply stable/prometheus-postgres-exporter/Chart.yaml]: Done!

**Comment by yashbhutwala** [comment]
@cpanato I made the fix you asked for.  Can you please review again and lgtm if possible?

File: stable/prometheus-postgres-exporter/templates/deployment.yaml [hunk 2]
```diff
@@ -48,6 +48,9 @@ spec:
           - {{ .Values.config.excludeDatabases | join "," }}
           {{- end }}
           {{- end }}
+{{- if .Values.config.extraArgs }}
+{{ toYaml .Values.config.extraArgs | indent 12 }}
```
Comment (davidkarlsen) [inline stable/prometheus-postgres-exporter/templates/deployment.yaml:52]: Shouldn't that be indent 10?

Comment (yashbhutwala) [reply stable/prometheus-postgres-exporter/templates/deployment.yaml:52]: you can template it out and test it, afaik, 12 is correct 😄

**Comment by scottrigby** [comment]
/hold

Let’s hold PRs to `stable` prometheus charts for a short time until prometheus-community/community#28 (comment) is resolved. If you want to help with that effort, to test etc, that would be great!

**Comment by scottrigby** [comment]
🚧 This chart is now deprecated.

Further development has moved to [prometheus-community/helm-charts](https://github.com/prometheus-community/helm-charts).

If applicable, please re-open this PR there. Thanks!

**Pull request closed without merging**